thirty-five-mm/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,71 @@
+class CommentsController < ApplicationController
+
+  def new
+    if params[:film_id]
+      @film = Film.find(params[:film_id])
+      @comment = Comment.new
+      render '_film_new_comment.html.erb', locals: { film: @film, comment: @comment }, layout: false
+    else
+      @review = Review.find(params[:review_id])
+      @comment = Comment.new
+      render '_review_new_comment.html.erb', locals: { review: @review, comment: @comment}, layout: false
+    end
+  end
+
+  def create
+    if params[:film_id]
+      @film = Film.find(params[:film_id])
+      @comment = @film.comments.build(comments_params)
+      @comment.user_id = User.first.id
```

Comment:
Are all your comments from the first user in your database. Maybe use current_user here?